PiwikTracker/PiwikTracker.swift

```diff
@@ -128,6 +128,14 @@ final public class PiwikTracker: NSObject {
     internal var visitor = Visitor.current()
     internal var session = Session.current()
     internal var nextEventStartsANewSession = true
+    
+    public func userAgent () -> String {
+        return self.dispatcher.userAgent!
+    }
+    public func setUserAgent (userAgent: String)
+    {
+        self.dispatcher.setUserAgent(ua: userAgent)
+    }
```

Comment:
Suggest to pass the user agent and the contractor and removing the possibly to change the user agent once from the tracker api.

If someone really really want to the ability the change the user agent after instantiation I suggest they do that by providing their own dispatcher implementation that support mutating the value.